app/static/src/app/components/run/RunStochasticPlot.vue

```diff
@@ -6,6 +6,9 @@
         :plot-data="allPlotData"
         :redrawWatches="solution ? [solution] : []"
         :linked-x-axis="linkedXAxis"
+        :fit-plot="false"
+        :graph-index="graphIndex"
+        :graph-config="graphConfig"
```

Comment:
We did have that. but as noted in the description, this caused reactivity issues:
> One other change is that the plots now take graphConfig as a prop rather than just the graph index - I was having a reactivity problem when graphs were deleted such that the last plot was still trying to read settings from the previous maximum indexed config in the store. Providing config as a prop solved this issue..

I'll actually put a comment in the code to this effect so we remember which we've done it this way.